server/apiCalls/transformers/memos.test.js

```diff
@@ -36,10 +42,16 @@ const offering_SF1625_202122 = {
   courseCode: 'SF1625',
   offeringId: '2',
   period: 'P1',
+  courseRoundApplications: [
+    {
+      course_round_application_code: '2',
+    },
+  ],
 }
 const memo_SF1625_202122_base = (courseCode = 'SF1625') => ({
   courseCode,
   ladokRoundIds: ['2'],
```

Comment:
`ladokRoundIds`  has to be removed to be sure tests working properly

Reply:
Removed